src/module/vue/sf-charactersheet.vue

```diff
@@ -108,18 +98,18 @@
   </div>
 </template>
 
-<style lang="less" scoped>
-.stat-roll {
-  text-transform: uppercase;
-}
-.tablist {
-  border-bottom: 1px solid grey;
-}
-.tab {
-  text-align: center;
-  padding: 5px;
-  &.active {
-    background-color: darkgray;
+<style lang="less">
```

Comment:
Are you not fond of `scoped`? It allows us to avoid having to write CSS classes just to scope styles in a single component.

Reply:
i'm not against `scoped` *per se* so much as `scoped` as a way to avoid CSS classes. here's my rationale on that:

1) i reckon the OuterHTML of *every* component -- plus any major elements within the component -- should have distinct, descriptive CSS classes in the first place to make the code easier to grok. this can be accomplished with comments, too, but adding a descriptive CSS class isn't really any more work.
2) with SFCs, identifying a specific element with a descriptive CSS class makes it easier to tell which styles in the template's `<style>` tag belong to which elements
3) inferring *why* an element is receiving a specific style becomes easier: "this is styled this way because scope 1234 specifies it" is less informative than "this is styled this way because it's a `.foo`", especially when an item is receiving styles from multiple selectors
4) finding where the heck the styling *happens* (especially with commonly-repeated properties) is easier if i can search for a specific class name. admittedly, less of a big deal the component CSS is exported as single files and can be inferred from that, though
5) beyond the semantics, the majority of html elements are just a box you can restyle in any number of ways... and often there's multiple ways of structuring the same element (though some are more semantically logical than others). descriptive css class names makes it easier to rewrite the html while keeping all the styling the same. so if i'm rewriting the `foo.vue`, and it has two things styled as buttons labelled "bar" and "baz", it's easier to rewrite that if i class its various widgets descriptively as `.foo-btn-bar` and `.foo-btn-baz` and target those for styling (and then figure out whether they need to be `<a>`, `<button>`, or something else entirely).

none of this precludes `scoped`, but if the above is happening anyways, it does make scoped less broadly applicable.